test/__snapshots__/reader.test.js.snap

```diff
@@ -7594,7 +7594,7 @@ exports[`should concat 1 file 1`] = `
             }
           }.call(
             this,
-            require(\\"/test/test-assets-reader/07a109ac983bc28d7f393215ca409e2d759e1fda9dc034e48338a5ef1aa92d6c/home/trygve/Work/component-prototype/micro-b-server/node_modules/process/browser.js\\"),
+            require(\\"../../../process/browser.js\\"),
```

Comment:
this has lost `node_modules` in the path, is that correct?

Reply:
yup, anytime a module in node_modules references another module in node_modules then node_modules will be lost from the path. Otherwise if will remain.